core/Command/SetupChecks.php

```diff
@@ -45,6 +45,31 @@ protected function configure(): void {
 		;
 	}
 
+	/**
+	 * @TODO move this method to a common service used by notifications, activity and this command
+	 * @throws \InvalidArgumentException if a parameter has no name or no type
+	 */
+	private function richToParsed(string $message, array $parameters): string {
+		$placeholders = [];
+		$replacements = [];
+		foreach ($parameters as $placeholder => $parameter) {
+			$placeholders[] = '{' . $placeholder . '}';
+			foreach (['name','type'] as $requiredField) {
+				if (!isset($parameter[$requiredField]) || !is_string($parameter[$requiredField])) {
+					throw new \InvalidArgumentException("Invalid rich object, {$requiredField} field is missing");
+				}
+			}
+			if ($parameter['type'] === 'user') {
+				$replacements[] = '@' . $parameter['name'];
+			} elseif ($parameter['type'] === 'file') {
+				$replacements[] = $parameter['path'] ?? $parameter['name'];
+			} else {
+				$replacements[] = $parameter['name'];
+			}
```

Comment:
We can use a `match` here: 
```suggestion
			$replacements[] = match($parameter['type']) {
				'user' => '@' . $parameter['name'],
				'file' => $parameter['path'] ?? $parameter['name'],
				default => $parameter['name'],
			}
```